### PR TITLE
[Feature]: Support get more field in status 

### PR DESCRIPTION
**Pull request by themanforfree**
Please briefly answer these questions:

Based on #548 

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)
Support get more field in status 
* what changes does this pull request make?
Support get more field in status 
* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)
no

### CODE REVIEW

File: curp/src/server/curp_node.rs [hunk 1]
```diff
@@ -869,6 +869,11 @@ impl<C: Command, RC: RoleChange> CurpNode<C, RC> {
         }
         Ok(())
     }
+
+    /// Get `RawCurp`
+    pub(super) fn curp(&self) -> Arc<RawCurp<C, RC>> {
```
Comment (bsbds) [inline curp/src/server/curp_node.rs:874]: Rename to `raw_curp` for consistency?

File: xline/src/server/maintenance.rs [hunk 2]
```diff
@@ -114,29 +134,34 @@ where
...
+        let last_applied = self.ce.last_applied().map_err(|e| {
+            error!("get last applied failed, {e}");
+            tonic::Status::internal("get last applied failed")
+        })?;
+        let mut errors = vec![];
+        if leader.is_none() {
+            errors.push("no leader".to_owned());
+        }
+        for a in self.alarm_store.get_all_alarms() {
+            errors.push(format!("{a:?}"));
```
Comment (bsbds) [inline xline/src/server/maintenance.rs:153]: Are the format of the errors consistent with etcd?

File: curp/src/server/curp_node.rs (hunk 1 above)
Comment (themanforfree) [reply curp/src/server/curp_node.rs]: Renamed

File: xline/src/server/maintenance.rs [hunk 3]
```diff
@@ -113,25 +129,34 @@ where
...
             version: env!("CARGO_PKG_VERSION").to_owned(),
-            db_size: -1,
-            leader: cluster.leader_id.unwrap_or(0), // None means this member believes there is no leader
-            raft_index: 0,
-            raft_term: cluster.term,
-            raft_applied_index: 0,
-            errors: vec![],
-            db_size_in_use: -1,
+            db_size: size.cast(),
+            leader: leader.unwrap_or(0), // None means this member believes there is no leader
```
[the remark this answers is not in the document]
Comment (themanforfree) [reply xline/src/server/maintenance.rs:154]: leader_id is 0 only if the node thinks there is no leader

**Review by Phoenix500526 — APPROVED** [review]

**Pull request merged**